src/components/list-box/_list-box.scss

```
@@ -291,3 +291,328 @@ $list-box-menu-width: rem(300px);
+    cursor: pointer;
+  }
+
+  .#{$prefix}--list-box__menu-icon > svg {
+    fill: $ui-05;
+    height: 100%;
+  }
+
+  .#{$prefix}--list-box__menu-icon--open {
+    transform: rotate(180deg);
+  }
+
+  // Selection indicator for a `list-box__field`
+  .#{$prefix}--list-box__selection {
+    display: inline-block;
+    position: absolute;
+    top: 0;
+    right: rem(26px);
+    bottom: 0;
+    height: rem(40px);
+    padding: 0 1rem;
+    cursor: pointer;
+    user-select: none;
+    transition: background-color 200ms $carbon--standard-easing;
+  }
+
+  .#{$prefix}--list-box__selection > svg {
+    fill: $ui-05;
+    height: 100%;
+  }
+
+  .#{$prefix}--list-box__selection:focus {
+    outline: 1px solid $brand-01;
+  }
+
+  // Modifier for a selection to show that multiple selections have been made
+  .#{$prefix}--list-box__selection--multi {
+    @include typescale('omega');
+
+    position: static;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0;
+    background-color: $brand-01;
+    height: rem(24px);
+    width: rem(42px);
+    color: white;
+    line-height: 0;
+    padding: 0.5rem;
+    margin-right: rem(10px);
+    border-radius: rem(12px);
+  }
+
+  .#{$prefix}--list-box__selection--multi > svg {
+    fill: white;
+    width: 0.5rem;
+    height: 0.5rem;
+    margin-left: rem(5px);
+  }
+
+  .#{$prefix}--list-box__selection--multi:focus,
+  .#{$prefix}--list-box__selection--multi:hover {
+    outline: none;
+  }
+
+  // Descendant of a `list-box` that displays a list of options to select
+  .#{$prefix}--list-box__menu {
+    @include box-shadow();
+
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: rem(32px);
+    width: $list-box-width;
+    background-color: $ui-01;
+    max-height: rem(120px);
+    overflow-y: auto;
+    z-index: z('dropdown');
+  }
+
+  // Descendant of a `list-box__menu` that represents a selection for a control
+  .#{$prefix}--list-box__menu-item {
+    @include typescale('zeta');
+
+    display: flex;
+    align-items: center;
+    height: rem(32px);
+    margin-top: -1px;
+    padding: 0 1rem;
+    cursor: pointer;
+    user-select: none;
+    border-bottom: 1px solid $ui-03;
+    position: relative;
+
+    &::before,
+    &::after {
+      content: '';
+      display: block;
+      position: absolute;
+      height: calc(100% + 1px);
+      width: 1rem;
+      background-color: $ui-01;
+    }
+
+    &::before {
+      left: 0;
+    }
+
+    &::after {
+      right: 0;
+    }
+
+    &:last-of-type {
+      border-bottom: 1px solid $ui-01;
+    }
+
+    &:last-of-type:hover {
+      border-bottom: 1px solid $ui-03;
+    }
+  }
+
+  .#{$prefix}--list-box__menu-item:hover,
+  .#{$prefix}--list-box__menu-item--highlighted {
+    background-color: $ui-03;
+    color: $text-01;
+    border-bottom: 1px solid $ui-01;
+
+    &::before,
+    &::after {
+      background-color: $ui-03;
+    }
+  }
+
+  .bx--list-box__menu-item .bx--checkbox-label {
```

Oh no! Looks like there are some lingering `bx` classes from the previous implementation 😬

---

Let's see if GH "change suggestion" feature works! 😉 

```suggestion
  .#${prefix}--list-box__menu-item .#${prefix}--checkbox-label {
```